### ISSUE TITLE
Removal of parking spaces is a bit flakey

### ISSUE THREAD

**Issue by originalfoo** [algernon-A/BOB#47]
Here's an Aldi supermarket that I've removed all parking spaces (set the various props, all three listed in screenshot, to 0% probability)...  Each time the game starts, the parking spaces seem to be retained somehow:

> ![image](https://user-images.githubusercontent.com/1386719/152916080-18e64033-0168-4a4d-82ee-4fe2dea5b8a9.png)
> https://steamcommunity.com/sharedfiles/filedetails/?id=447763113

If I reset them then put them back to 0% probability they stay gone for the remainder of the game, but will suddenly be back again on next session.

I noticed that Klyte's Prop Replacer mod that I used to use had similar issues with parking spaces. Maybe they get cached somewhere eg. a parking grid or something?

**Comment by algernon-A** [comment]
Think I see where the issue might be - if I'm right, it's more to do with a particular building prefab flag; will investigate the best way to resolve it.

**Comment by algernon-A** [comment]
Having troubles reproducing the exact issue, though - is it that the probabilities are resetting (to not zero), or that the probabilities are indeed zero but cars are still parking?

**Comment by originalfoo** [comment]
Probabilities were zero (according to BOB UI) but cars were still parking.

After changing probability to non-zero, saving (green tick button), then setting back to zero again, it seemed to get rid of the parking space for that session - however I'd have to change something nearby (eg. use moveit to move a tree or whatever) to force a parking reshuffle at which point that parked vehicle in image realised it no longer had parking space and thus teleported somewhere else. For the remainder of that session, vehicles would not park there.

So the issue seems to be when a savegame is loaded, the parking spaces aren't entirely removed (or are cached somewhere or whatever) and it's only after I redo probabilities in BOB that no more vehicles will park there for that play session.
